provider/aws/dist/rack.json

```diff
@@ -1249,7 +1270,6 @@
       "Properties": {
         "GroupDescription": "Instances",
         "SecurityGroupIngress": [
-          { "IpProtocol": "tcp", "FromPort": "22", "ToPort": "22", "CidrIp": { "Ref": "VPCCIDR" } },
```

Comment:
This came with the dedicated build changes: https://github.com/convox/rack/pull/1677/files#diff-54ede314b4e7f9fbc11373ad5f9b8bc2L1252